src/App.css

```diff
@@ -291,135 +291,216 @@
 /* Board Layout */
 .board {
   display: flex;
-  gap: 24px;
+  gap: 16px;
   overflow-x: auto;
-  padding-bottom: 16px;
+  padding: 16px;
+  align-items: flex-start;
+  height: calc(100vh - 90px);
 }
 
 /* Column Styles */
 .column {
   background: var(--column-background);
-  border-radius: 8px;
+  border-radius: 12px;
+  width: 300px;
   min-width: 300px;
-  padding: 16px;
-  box-shadow: 0 1px 2px rgba(0, 0, 0, 0.1);
-  border: 1px solid var(--border-color);
-  transition: background-color 0.3s ease;
+  max-height: 100%;
+  display: flex;
+  flex-direction: column;
+  box-shadow: var(--ds-shadow-raised,0 1px 1px rgba(9,30,66,0.25));
+  transition: box-shadow 0.2s ease;
```

Comment:
![kody code-review](https://img.shields.io/badge/kody-code--review-brightgreen)

```javascript
:root {
  --transition-fast: 0.1s;
  --transition-medium: 0.2s;
  --transition-slow: 0.3s;
}

.element {
  transition: all var(--transition-medium) ease;
}
```
Consider using CSS custom properties for transition durations to maintain consistency and enable easier updates. Multiple elements use '0.2s' or '0.3s' for transitions.



<sub>Talk to Kody by mentioning @kody</sub>

<sub>Was this suggestion helpful? React with 👍 or 👎 to help Kody learn from this interaction.</sub><!-- kody-codereview -->&#8203;
&#8203;